### DISCUSSION TITLE
Recommended already starred repository

### DISCUSSION THREAD

**Discussion by Reslan-Tinawi** [community/community#14381] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello GitHub team,

First, I'd like to say: great work on the new feed 🎉 

I noticed something that I think is a bit confusing (or can be considered a bug).

In the feed, I had a repository under the recommended for you, although I already knew and starred the repository.

The repository is: [gboeing/osmnx](https://github.com/gboeing/osmnx).

You can also see it in the attached screenshot.

![GitHub](https://user-images.githubusercontent.com/9997649/161959217-f7f7a4e4-ec72-46f5-8c5b-eb8d428c38a8.png)

Thanks!

**Comment by jfullstackdev** [comment, marked as answer]
just want to mention this, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
